src/Bridge/Symfony/Bundle/Command/GraphQlExportCommand.php

```diff
@@ -79,4 +77,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         return 0;
     }
+
+    // Use AsCommand Attribute when SF 4.4 and PHP 7 gets dropped
```

Comment:
In my opinion you can even drop this comment. There is no benefits to use an attribute here and it will be a small BC break. Same for other classes.

Reply:
Yeah, just thought I'd mention it in case the way commands are registered changes in the future for some reason (or if something needs to be added). I can remove them though